### PR TITLE
Added script to monitor failed queue and send emails.

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
monitors failed queue and sends email. Currently scheduled to run in quiet mode every 3 hrs, and in verbose mode at noon.

### CODE REVIEW

File: share/bash/checkfailures.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+quiet=$1
+status=$(bin/console messenger:failed:show 2>&1)
+errormsg=$(echo $status | grep 'There are 0 messages pending in the failure transport.')
+
+if [ "$quiet" == "quiet" ]; then
+    if [ -z "$errormsg" ]; then
+        echo $status | /bin/mailx -s "Failed message transport status" michael.williamson@tamucc.edu praneeth.pondicherry@tamucc.edu michael.vandeneijnden@tamucc.edu william.nichols@tamucc.edu rosalie.rossi@tamucc.edu
```
Comment (praneethpr) [inline share/bash/checkfailures.sh:9]: is it okay to hardcode our emails to the git repo?

Comment (mickel1138) [reply share/bash/checkfailures.sh:9]: Yes, don't like it!

Comment (mickel1138) [reply share/bash/checkfailures.sh:9]: Also, can the subject NOT contain the word Failed, when I see it I think something failed.

**Review by praneethpr — APPROVED** [review]
LGTM!

**Comment by mickel1138** [comment]
needs pelagos ticket!

**Comment by mickel1138** [comment]
Re-open when there is a ticket for this, and PO blessed it.

**Pull request closed without merging**